config.json

```diff
@@ -16,6 +16,7 @@
 
   ],
   "exercises": [
+    { "difficulty" : 1, "slug" : "allergies", "topics" : [] },
```

Comment:
I was going for alphabetical order so that there would be no merge conflict when it's merged into master. But okay, I'll move it at the end. You deal with the conflict, though. :-)